src/MerchantCenter/MerchantStatuses.php

```diff
@@ -811,6 +811,12 @@ protected function parse_presync_issue_text( string $text ): array {
 		if ( 'imageLink' === $matches[1] ) {
 			$matches[1] = 'image';
 		}
+
+		// Convert additionalImageLinks[] to galleryImage.
```

Comment:
💅  Was confusing for me at first glance this comment. AdditionalImageLinks[] is syntactically an array and I thought we were converting some kind of variable data into other format. After reading the context I understood that `Convert additionalImageLinks[] to galleryImage` means `replace "additionalImageLinks[]" in the issue description to "galleryImage"`

Same goes with the old comment in line 810